site/sitecontent/about.md

```diff
@@ -9,3 +9,39 @@ Towards this goal, LibreCores provides you
 
 LibreCores is a project of the [Free and Open Source Silicon Foundation](http://www.fossi-foundation.org), short FOSSi.
 
+## Frequently Asked Questions (FAQ)
+
+### Who's running LibreCores?
+LibreCores is a project of the [Free and Open Source Silicon Foundation](http://www.fossi-foundation.org) (FOSSi), which was created to give a voice to the digital hardware design community. LibreCores, like FOSSi, is run entirely by volunteers.
+
+### How does LibreCores relate to OpenCores?
+LibreCores advances the idea of OpenCores.org to give the community a place to share projects, ideas, and knowledge in the area of free and open source digital hardware design.
+
+As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our power to evolve the concept together with the creators of OpenCores. Unfortunately, to this day, we were not yet able to achieve this goal.
+Therefore, we were forced to use the LibreCores brand instead for the time being.
```

Comment:
I would formulate slightly different: "Therefore, we were forced to use the LibreCores brand instead for the time being." -> "Therefore, we decided to give it a real fresh start under the LibreCores brand."